ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-device-documentation.j2

```diff
@@ -179,6 +179,10 @@
 {% include 'documentation/mac-security.j2' %}
 {# Traffic Policies #}
 {% include 'documentation/traffic-policies.j2' %}
+{# AS-Path access-lists #}
```

Comment:
I think this section should be moved up under "IPv6 Extended Access-lists". Also include two `##` so it is clear that it is a subheading that goes under `# ACL`.